data/mms-published-branches.yaml

```diff
@@ -15,17 +17,21 @@ version:
     - '1.1'
   active:
     - 'upcoming'
+    - '4.3 (rapid)'
+    - '4.2'
     - '4.1 (rapid)'
     - '4.0'
     - '3.6'
     - '3.4'
-  stable: '4.0'
-  upcoming: '4.2'
+  stable: '4.2'
+  upcoming: '4.4'
```

Comment:
Should this be `upcoming`? If it doesn't match one of the entries in the `active` list, it won't do anything

Reply:
That does not appear to be correct. The upcoming _now_ says `4.2` and works. Is there something I'm missing?

Reply:
Ah, I see. You're *not actually using* this field on opsmgr.

This field corresponds to an entry in the `active` list to get a ` (upcoming)` suffix. But because you actually named the version `upcoming`, this field should never be used because otherwise `upcoming (upcoming)` would be rendered.

lgtm because this is a trivial concern that doesn't need to hold up anything :+1: